### PR TITLE
Railsify app

### PR DESCRIPTION
**Pull request by ghost**
Convert the humble app to rails.

### CODE REVIEW

**Comment by adamnoto** [comment]
tolong di review yah wen

File: app/assets/javascripts/home.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (wendy0402) [inline app/assets/javascripts/home.coffee]: let's just use javascript, delete this

File: config/application.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+require File.expand_path('../boot', __FILE__)
+
+require 'rails/all'
+
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
+Bundler.require(*Rails.groups)
+
+module System2
```
Comment (wendy0402) [inline config/application.rb]: I thought we gonna name it shopoza?

**Comment by wendy0402** [comment]
dude, no need to put bootstrap in vendor, just use external cdn link, faster

**Comment by wendy0402** [comment]
can you rename the branch with step 0-configuring rails? how about that? so when they see the branch, they can understand immediately

File: app/assets/javascripts/home.coffee (hunk 1 above)
Comment (adamnoto) [inline app/assets/javascripts/home.coffee]: yep. I will change it to .js and give explanation a bit what's coffeescript and why we stick with js.

File: config/application.rb (hunk 2 above)
Comment (adamnoto) [inline config/application.rb]: will change

**Comment by adamnoto** [comment]
@wendy0402 but then again, not all readers will have stable internet connection. better bundle it with the app, I think.

nice idea! will change

**Pull request closed without merging**